app.js

```diff
@@ -43,12 +45,24 @@ client.connect()
     })
   });
 
+<<<<<<< HEAD
+
+  //function render bucketlistResultaat page
+function showBucketlistResultaat(req, res) {
+    res.render('pages/bucketlist/bucketlistResultaat', {title: 'bucketlistoverzicht'}, {interestView: data});
+  };
+
+function saveBucketlist(req, res) { 
+  // here we will save the bucketlist later on
+}
+=======
   app.get('/add', profileForm);
   app.post('/add', upload.single('photo'), add);
   
   //bucketlist
   app.get('/bucketlist', showBucketlistOverview);
   app.get('/bucketlistResults', showBucketlistResults);
+>>>>>>> develop
```

Comment:
Make sure you don't save and commit merge conflicts. Note that there are two parts one name `HEAD` and one named `develop`. Remove the part of the code you don't need which could be either `HEAD`, `develop` or a mixture of both. Remove any of these lines (also including the separator line at 58) that don't contribute, and commit and push this.

In future, when you're encountering merge conflict, your code editor should provide you with the options "Accept current changes", "Accept Incoming changes", "Accept both". Pick one, and don't forget to save! Otherwise you'll commit and push these merge messages.